operations/observability/mixins/cross-teams/dashboards.libsonnet

```diff
@@ -17,5 +13,9 @@
     'gitpod-node-resource-metrics.json': (import 'dashboards/gitpod-node-resource-metrics.json'),
     'gitpod-grpc-server.json': (import 'dashboards/gitpod-grpc-server.json'),
     'gitpod-grpc-client.json': (import 'dashboards/gitpod-grpc-client.json'),
+    'gitpod-overview.json': (import 'dashboards/gitpod-overview.json'),
+    'gitpod-nodes-overview.json': (import 'dashboards/gitpod-nodes-overview.json'),
+    'gitpod-admin-node.json': (import 'dashboards/gitpod-admin-node.json'),
+    'gitpod-admin-workspace.json': (import 'dashboards/gitpod-admin-workspace.json')
```

Comment:
silly question: do you need a comma at the end here? (some were breaking in the past for it, not sure if this is the case here or not)

Reply:
Since all dashboards are present in the preview, I believe there's no need :)